apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts

```diff
@@ -310,6 +314,12 @@ export class PurchaseRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true,
             },
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
+            if (activeCompany) {
+                this.activeCompanyCountryCode = activeCompany.countryV2?.alpha2CountryCode;
```

Comment:
confirm with @ravindersinghdang sir for not using component store for this or not

Reply:
Done